android/build_boinc_arm.sh

```diff
@@ -51,7 +51,9 @@ if [ -n "$COMPILEBOINC" ]; then
 
     echo "Stripping Binaries"
     cd stage/usr/local/bin
-    arm-linux-androideabi-strip *
+    if [ ! "$BOINC_DEBUG" = "ON" ]; then
```

Comment:
Let's use general `DEBUG`. I see no reason why it should have some side effects.